src/diffusers/pipelines/stable_diffusion_xl/pipeline_stable_diffusion_xl.py

```diff
@@ -545,6 +564,8 @@ def __call__(
         height: Optional[int] = None,
         width: Optional[int] = None,
         num_inference_steps: int = 50,
+        final_inference_step: Optional[int] = None,
```

Comment:
```suggestion
```

Reply:
still needed to override this for more than dual stage pipelines, eg if you wanted to do 2 steps here, 2 steps there, alternating back and forth